### PR TITLE
The removed quote

### PR DESCRIPTION
**Pull request by Andrei-Predoiu**
https://bestjira.atlassian.net/browse/ES-1186

### CODE REVIEW

File: secrets/format.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func (result Result) toKV(prefix string) string {
 	for key, val := range result {
 		leKey := fixEnvName(key)
 		fmt.Println(leKey)
-		resturnString += fmt.Sprintf("%s%s='%s'\n", prefix, leKey, val)
+		resturnString += fmt.Sprintf("%s%s=%s\n", prefix, leKey, val)
```
Comment (brondum) [inline secrets/format.go:35]: This change will fuck up the ENV part as it uses the to kv as well.

Comment (Andrei-Predoiu) [reply secrets/format.go:35]: Alright, i'll try something else

Comment (Gaardsholt) [reply secrets/format.go:35]: hmm.. if I run `GITHUB_TOKEN='1234'` in my terminal and then do a `echo $GITHUB_TOKEN` the single quotes is not printed?

Comment (brondum) [reply secrets/format.go:35]: Nope, it strips the single quotes are stripped by the export command, but it uses them when you parses strings containing special chars.

**Review by brondum — APPROVED** [review]
LGTM

**Pull request merged**